paddle/fluid/framework/new_executor/standalone_executor_test.cc

```diff
@@ -137,29 +137,29 @@ ProgramDesc GetLmMainProgram() {
   return main_prog;
 }
 
-TEST(StandaloneExecutor, run) {
-  auto place = platform::CUDAPlace(0);
-  ProgramDesc test_prog = load_from_file("lm_startup_program");
-  ProgramDesc main_prog = GetLmMainProgram();
+// TEST(StandaloneExecutor, run) {
```

Comment:
Why comment out this test case?

Reply:
By mistake, it will be opened in the next PR.